helm/ffc-pay-web/values.yaml

```diff
@@ -47,6 +47,10 @@ container:
   azureStorageUseConnectionString: false
   azureStorageCreateContainers: false
   azureStorageAccountName: ffcpay
+  useV2Events: true
```

Comment:
Am just gonna reference my question here: https://github.com/DEFRA/ffc-pay-web/pull/48/files#r1166793031

Reply:
Yeah, we want this to be the default.